dags/synapse-dataset-to-croissant.py

```diff
@@ -381,37 +356,55 @@ def construct_distribution_section_for_files(files_attached_to_dataset: List[Fil
         SELECT
             vd.id,
             vd.versionNumber,
-            vd.contentMd5
+            vd.contentMd5,
+            vd.fileHandleId,
+            COALESCE(fl.content_type, 'NOT_SET') as CONTENT_TYPE,
         FROM version_data AS vd
+        LEFT JOIN synapse_data_warehouse.synapse.file_latest fl
+            ON fl.id = vd.fileHandleId
```

Comment:
`CONTENT_TYPE` is not available within the `VERSION_HISTORY` column, which is why there is a need here to do a join to the `file_latest` table.